### DISCUSSION TITLE
Application Icon always cropped

### DISCUSSION THREAD

**Discussion by SergTomcat** [dotnet/maui#15160] [Q&A]
I'm trying to add app logo. I have a square logo that needed to be shown full, so basically inscribed in OS icon circle. But it always crops.
I was trying manually add empty area with regarding of crop (red zone) 

![image](https://github.com/dotnet/maui/assets/24654961/57e2a45f-98ae-4d94-8add-911a3d186b5f)

but it still getting cropped:
![image](https://github.com/dotnet/maui/assets/24654961/715c6cd6-0917-4343-b62f-48f4a24e0a0c)

Tried to play with "BaseSize" and "Resize" attributes, still no success.

Please, explain how to properly add app logo, because nothing special mentioned in [docs](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/images/app-icons?tabs=windows).

SVG:

&lt;svg width=&quot;230&quot; height=&quot;230&quot; xmlns=&quot;http://www.w3.org/2000/svg&quot;&gt;
 &lt;defs&gt;
  &lt;clipPath id=&quot;clip0_1354_68684&quot;&gt;
   &lt;rect id=&quot;svg_1&quot; fill=&quot;white&quot; height=&quot;230&quot; width=&quot;230&quot;/&gt;
  &lt;/clipPath&gt;
 &lt;/defs&gt;
 
 &lt;g&gt;
	 &lt;rect fill=&quot;red&quot; id=&quot;canvas_background&quot; height=&quot;232&quot; width=&quot;232&quot; y=&quot;-1&quot; x=&quot;-1&quot;/&gt;
  &lt;title&gt;Layer 1&lt;/title&gt;
  &lt;rect x=&quot;34&quot; id=&quot;svg_3&quot; fill=&quot;#00FF00&quot; height=&quot;53.3333&quot; width=&quot;53.3333&quot; y=&quot;140.666&quot;/&gt;
  &lt;rect id=&quot;svg_4&quot; fill=&quot;#0000FF&quot; height=&quot;53.3333&quot; width=&quot;53.3333&quot; y=&quot;140.666&quot; x=&quot;87.3333&quot;/&gt;
  &lt;rect id=&quot;svg_5&quot; fill=&quot;white&quot; height=&quot;53.3333&quot; width=&quot;53.3333&quot; y=&quot;140.666&quot; x=&quot;140.667&quot;/&gt;
  &lt;rect x=&quot;34&quot; id=&quot;svg_6&quot; fill=&quot;#0000FF&quot; height=&quot;53.3333&quot; width=&quot;53.3333&quot; y=&quot;87.334&quot;/&gt;
  &lt;rect id=&quot;svg_7&quot; fill=&quot;#FFFF00&quot; height=&quot;53.3333&quot; width=&quot;53.3333&quot; y=&quot;87.334&quot; x=&quot;87.3333&quot;/&gt;
  &lt;rect id=&quot;svg_8&quot; fill=&quot;#4176EA&quot; height=&quot;53.3333&quot; width=&quot;53.3333&quot; y=&quot;87.334&quot; x=&quot;140.667&quot;/&gt;
  &lt;rect x=&quot;34&quot; y=&quot;34&quot; id=&quot;svg_9&quot; fill=&quot;white&quot; height=&quot;53.3333&quot; width=&quot;53.3333&quot;/&gt;
  &lt;rect y=&quot;34&quot; id=&quot;svg_10&quot; fill=&quot;#4176EA&quot; height=&quot;53.3333&quot; width=&quot;53.3333&quot; x=&quot;87.3333&quot;/&gt;
  &lt;rect y=&quot;34&quot; id=&quot;svg_11&quot; fill=&quot;#DEE9FF&quot; height=&quot;53.3333&quot; width=&quot;53.3333&quot; x=&quot;140.667&quot;/&gt;
 &lt;/g&gt;
&lt;/svg&gt;

**Comment by mhrastegari** [comment]
Add `ForegroundScale="0.85"` (or whatever size you want) to the `MauiIcon` tag in your `csproj`.
